### PR TITLE
Remove domain specific references

### PR DESCRIPTION
**Pull request by DirkDevelopmentFreelance**

### CODE REVIEW

**Comment by DirkDevelopmentFreelance** [comment]
Should I remove all domain specific references inside tailwind css and classes?

File: api/.env.example [hunk 1]
```diff
@@ -1,5 +1,11 @@
 PORT=3000
 
+ORIGIN=[
```
Comment (ivanslabbert) [inline api/.env.example:3]:
```suggestion
CORS_ORIGIN=[
```

File: api/src/main.ts [hunk 2]
```diff
@@ -31,12 +31,7 @@ async function bootstrap() {
     await upgradeDbSchema(dbService, s3Service);
 
     app.enableCors({
-        origin: [
-            "http://localhost:4174",
-            "http://localhost:4175",
-            "https://app2.bcc.africa",
-            "https://admin.app2.bcc.africa",
-        ],
+        origin: process.env.ORIGIN,
```
Comment (ivanslabbert) [inline api/src/main.ts:34]:
```suggestion
        origin: process.env.CORS_ORIGIN,
```

File: app/.env.example [hunk 3]
```diff
@@ -1,4 +1,6 @@
 VITE_APP_NAME="Luminary App"
+VITE_BRAND_NAME="Some brand name"
+VITE_LOGO_SVG='<svg></svg>'
```
Comment (ivanslabbert) [inline app/.env.example:3]: Change to VITE_LOGO and VITE_LOGO_SMALL. Add comments in the example env file to explain the different uses. The variable value should be the relative path or absolute http link to the file.

File: app/src/components/LLogo.vue [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<script setup lang="ts">
+const logo = import.meta.env.VITE_LOGO_SVG;
+</script>
+
+<template>
+    <div v-html="logo"></div>
+</template>
```
Comment (ivanslabbert) [inline app/src/components/LLogo.vue:7]: This component can be removed and logo implemented directly in TopBar.vue

File: app/src/components/navigation/TopBar.vue [hunk 5]
```diff
@@ -6,6 +6,7 @@ import DesktopMenu from "./DesktopMenu.vue";
 
 const route = useRoute();
 const router = useRouter();
+const BRAND_NAME = import.meta.env.VITE_BRAND_NAME;
```
Comment (ivanslabbert) [inline app/src/components/navigation/TopBar.vue:9]: Rather use logos instead of text based brand name

File: app/src/pages/LoginPage.vue [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <script setup lang="ts">
 import LButton from "@/components/button/LButton.vue";
 import LCard from "@/components/common/LCard.vue";
-import BccLogo from "@/components/BccLogo.vue";
+import LLogo from "@/components/LLogo.vue";
```
Comment (ivanslabbert) [inline app/src/pages/LoginPage.vue:4]: Do not change this yet (keep the original BCC logo). This will be updated when the Auth0 integration is redone

**Review by ivanslabbert — CHANGES_REQUESTED** [review]

**Pull request merged**